### PR TITLE
Download buttons

### LINKED ISSUE #41: Add download buttons for Summary and Transcribe
- [ ] Default the filename to `<uploaded filename>-summary.txt`  `<uploaded filename>-transcription.txt`

### PR DESCRIPTION
**Pull request by Noxsios**
Closes #41

### CODE REVIEW

**Comment by Noxsios** [comment]
I did as much verification / testing as I could w/o access to LFAI, please test the 2 new download buttons when you can.

@justinthelaw @CollectiveUnicorn

File: manifests/deploy.yaml [hunk 1]
```diff
@@ -24,8 +24,8 @@ spec:
               value: "https://###ZARF_VAR_DOMAIN###"
           resources:
             requests:
-              memory: "64Mi"
+              memory: "500Mi"
               cpu: "500m"
             limits:
-              memory: "128Mi"
+              memory: "800Mi"
```
Comment (justinthelaw) [inline manifests/deploy.yaml:30]: I think for max memory, just to be safe, we should lean towards 1.5Gi. Reason being, I tested a 1 hour audio and it took up nearly 1Gi of RAM (an OpenAI library issue), so I upped the container memory limit accordingly. I still need to test on a larger sample size, but the short term fix is to just allocate more memory. It's possible some users will need to transcribe longer meetings or possibly concurrently process multiple shorter meetings.

In the long term, we may want to do our own client-side chunking and reassembly of the audio. For now, I think we just stick with more memory limit while we think of a better solution.

**Review by justinthelaw — CHANGES_REQUESTED** [review]
Overall looks great! I think the change to memory should be addressed in a different PR (for separation of concerns purposes), which I can push today on a branch I was on.

**Review by CollectiveUnicorn — APPROVED** [review]

**Review by CollectiveUnicorn — CHANGES_REQUESTED** [review]
I'm getting a weird artifact on the page with a large transcript

**Comment by justinthelaw** [comment]
@Noxsios @CollectiveUnicorn I also get the same visual artifact. 
image

**Review by justinthelaw — APPROVED** [review]
Looks good to me! I think with the time constraint, this is good enough.

**Review by CollectiveUnicorn — APPROVED** [review]

**Pull request merged**